### PR TITLE
code format

### PR DESCRIPTION
**Pull request by wangnimaa**
## What is the purpose of the change

XXXXX

## Brief changelog

XXXXX

## Verifying this change

XXXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a [GITHUB_issue](https://github.com/apache/incubator-dubbo/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a GITHUB issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Format the pull request title like `[Dubbo-XXX] Fix UnknownException when host config not exist #XXX`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/incubator-dubbo/tree/master/dubbo-test).
- [ ] Run `mvn clean install -DskipTests` & `mvn clean test-compile failsafe:integration-test` to make sure unit-test and integration-test pass.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/incubator-dubbo/wiki/Software-donation-guide).

### CODE REVIEW

**Comment by diecui1202** [comment]
@wangnimaa This PR's code format is not adopted by Dubbo, check https://github.com/apache/incubator-dubbo/blob/master/codestyle/dubbo_codestyle_for_idea.xml

**Comment by wangnimaa** [comment]
> @wangnimaa This PR's code format is not adopted by Dubbo, check https://github.com/apache/incubator-dubbo/blob/master/codestyle/dubbo_codestyle_for_idea.xml

用这个没修改的地方也报错
image

**Comment by wangnimaa** [comment]
> @wangnimaa This PR's code format is not adopted by Dubbo, check https://github.com/apache/incubator-dubbo/blob/master/codestyle/dubbo_codestyle_for_idea.xml

**Comment by diecui1202** [comment]
It works fine at our env. You could check u IDEA configurations.

**Comment by wangnimaa** [comment]
I submit code again

**Comment by wangnimaa** [comment]
> It works fine at our env. You could check u IDEA configurations.

ok,I submit code again

File: dubbo-common/src/main/java/org/apache/dubbo/common/bytecode/ClassGenerator.java [hunk 1]
```diff
@@ -46,288 +47,345 @@
...
-    private static String modifier(int mod) {
-        StringBuilder modifier = new StringBuilder();
-        if (Modifier.isPublic(mod)) modifier.append("public");
-        if (Modifier.isProtected(mod)) modifier.append("protected");
-        if (Modifier.isPrivate(mod)) modifier.append("private");
-
-        if (Modifier.isStatic(mod)) modifier.append(" static");
-        if (Modifier.isVolatile(mod)) modifier.append(" volatile");
+  private static final AtomicLong CLASS_NAME_COUNTER = new AtomicLong(0);
+  private static final String SIMPLE_NAME_TAG = "<init>";
```
Comment (diecui1202) [inline dubbo-common/src/main/java/org/apache/dubbo/common/bytecode/ClassGenerator.java]: This is not the code format we need.

Comment (diecui1202) [reply dubbo-common/src/main/java/org/apache/dubbo/common/bytecode/ClassGenerator.java]: use 4 blanks instead.

Comment (wangnimaa) [reply dubbo-common/src/main/java/org/apache/dubbo/common/bytecode/ClassGenerator.java]: image
but he told me 2 blanks

Comment (diecui1202) [reply dubbo-common/src/main/java/org/apache/dubbo/common/bytecode/ClassGenerator.java]: what is the plugin you used? It's all ok on my mac.

Comment (wangnimaa) [reply dubbo-common/src/main/java/org/apache/dubbo/common/bytecode/ClassGenerator.java]: I have git hook，it's conflict

**Comment by diecui1202** [comment]
LGTM.

**Pull request merged**